l10n_it_account_stamp/models/company.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from openerp import fields, models, api
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    tax_stamp_product_id = fields.Many2one(
+        'product.product', 'Tax Stamp Product',
+        help="Product used to model DatiBollo XML element on bills."
```

Comment:
Essendo un modulo indipendente dalla fattura elettronica, l'XML non centra.
Metterei `Product used as Tax Stamp in customer invoices`

Reply:
fatto